### PR TITLE
(DOCSP-43903): Update local dev setup instructions

### PR DESCRIPTION
**Pull request by davidhou17**
Updates prerequisites to more closely match the [vector search local dev](https://www.mongodb.com/docs/atlas/atlas-vector-search/tutorials/vector-search-quick-start) procedure.

- [DOCSP-43903](https://jira.mongodb.org/browse/DOCSP-43903)
- [STAGING](https://deploy-preview-757--docs-atlas-cli.netlify.app/atlas-cli-deploy-local/)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [x] Resolve any new warnings or errors in the build.
- [x] Proofread for spelling and grammatical errors.
- [x] Check staging for rendering issues.
- [x] Confirm links are working.

### CODE REVIEW

File: source/includes/steps-atlas-cli-deploy-local-prereqs.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the dependencies.
+
+      a. Install the :ref:`{+atlas-cli+} <install-atlas-cli>`, 
+         :dbtools:`MongoDB Database Tools </installation/installation/>`,
```
Comment (jeroenvervaeke) [inline source/includes/steps-atlas-cli-deploy-local-prereqs.rst:8]: Afaik `mongosh` is optional when you decide to use `compass` instead (both are optional).

Comment (jeroenvervaeke) [reply source/includes/steps-atlas-cli-deploy-local-prereqs.rst:8]: Maybe this could be a section like `compass` section

Comment (davidhou17) [reply source/includes/steps-atlas-cli-deploy-local-prereqs.rst]: Awesome, thanks @jeroenvervaeke and @fmenezes, made those updates and this should be ready for another look. If this looks good, I'll also apply this to our vector search documentation separately.

**Review by fmenezes — APPROVED** [review]

File: source/includes/steps-atlas-cli-deploy-local-prereqs.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      a. Install the :ref:`{+atlas-cli+} <install-atlas-cli>`.
+
+         If you use `Homebrew <https://brew.sh/#install>`__, you can
+         run the following command in your terminal:
+
+         .. code-block::
+
+            brew install mongodb-atlas-cli
+               
+         For install instructions on other operating systems,
```
Comment (JuliaMongo) [inline source/includes/steps-atlas-cli-deploy-local-prereqs.rst:16]: installation

File: source/includes/steps-atlas-cli-deploy-local-prereqs.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+         - For MacOS or Windows, install `Docker Desktop v4.31+ <https://docs.docker.com/desktop/release-notes/#4310>`__. 
+         - For Linux, install `Docker Engine v27.0+ <https://docs.docker.com/engine/release-notes/27.0/>`__.
+
+         .. note::
+
+            Docker requires a network connection for pulling and caching 
+            MongoDB images.
+
+            `Podman v5.0+ <https://podman.io>`__ is also supported for Linux RHEL versions.
```
Comment (JuliaMongo) [inline source/includes/steps-atlas-cli-deploy-local-prereqs.rst:29]: For RHEL, you can also use |podman|.

Suggestion: Add Podman, Docker and Homebrew to the snooty.toml replacements. If you do, replace them in this file with |homebrew|, |docker|, and |podman|.

File: source/includes/steps-atlas-cli-deploy-local-prereqs.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+            `Podman v5.0+ <https://podman.io>`__ is also supported for Linux RHEL versions.  
+
+      #. (Optional) Install :mongosh:`mongosh </install>` version 2.0 or later.
+
+         .. code-block:: sh
+
+            brew install mongosh
+
+         For install instructions on other operating
```
Comment (JuliaMongo) [inline source/includes/steps-atlas-cli-deploy-local-prereqs.rst:37]: installation

File: source/includes/steps-atlas-cli-deploy-local-prereqs.rst [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+         For install instructions on other operating 
+         systems, see :mongosh:`Install mongosh </install>`.
+         
+      #. (Optional) Install :compass:`Compass </install>` version 1.39.4 or later.
+
+         .. code-block:: sh
+
+            brew install mongodb-compass
+
+         For install instructions on other operating
```
Comment (JuliaMongo) [inline source/includes/steps-atlas-cli-deploy-local-prereqs.rst:46]: installation

File: source/atlas-cli-deploy-local.txt [hunk 6]
```diff
@@ -74,7 +74,14 @@ Supported OS for Local Atlas Deployments
 Complete the Prerequisites
 --------------------------
 
-.. include:: /includes/fact-deploy-tutorial-prereqs.rst
+Before you begin, complete the following prerequisites:
+
+.. important::
+
+   For compatibility information for each prerequisite product, see the
```
Comment (JuliaMongo) [inline source/atlas-cli-deploy-local.txt:81]: Suggestion: 

For compatibility information on each product in the dependencies list, see the product's installation documentation.

File: source/includes/steps-atlas-cli-deploy-local-prereqs.rst [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+               
+         For install instructions on other operating systems,
+         see :ref:`install-atlas-cli`.
+
+      #. Install `Docker <https://www.docker.com/>`__.
+
+         - For MacOS or Windows, install `Docker Desktop v4.31+ <https://docs.docker.com/desktop/release-notes/#4310>`__. 
+         - For Linux, install `Docker Engine v27.0+ <https://docs.docker.com/engine/release-notes/27.0/>`__.
+
+         .. note::
```
Comment (JuliaMongo) [inline source/includes/steps-atlas-cli-deploy-local-prereqs.rst:24]: I would get rid of the note and move the first sentence about Docker to be right after the sentence on line 19 (where the step is). 
I would then move the second sentence to be the third bullet.

**Review by JuliaMongo — APPROVED** [review]
LGTM with a few small copy review suggestions! Thank you, David!

File: source/includes/steps-atlas-cli-deploy-local-prereqs.rst (hunk 7 above)
Comment (davidhou17) [reply source/includes/steps-atlas-cli-deploy-local-prereqs.rst:24]: Great suggestion!

**Pull request merged**